### PR TITLE
Tab bar has one vertical line on the left side and no line on right side

### PR DESCRIPTION
**Pull request by monikamaheshwari**
Issue #47

### CODE REVIEW

**Comment by glawrence** [comment]
I have tested this and it works for me, it also works fine with #36 so I have tested this twice. I also like the change, so thank you Monika

**Comment by april** [comment]
The side effect of this is that the border between each tab is twice as think as the outer border. Could you collapse them?

**Comment by april** [comment]
Namely, if you look at the generated HTML, you have one tab separator on the outside and two separators between each of the other tabs. These should probably just be collapsing borders instead.

**Comment by monikamaheshwari** [comment]
does collapsing border means you don't want border on both the outer sides ?

**Comment by april** [comment]
No, it means that the border between tabs is 2px wide and the border on the outside is 1px. Those inside borders should all be 1px, and honestly we probably shouldn't be using elements as dividers for this anyways.  :)

**Comment by monikamaheshwari** [comment]
ya right. Will change that :)

**Comment by glawrence** [comment]
I have tested and visually looks good now

**Comment by april** [comment]
This looks great except for the change to the handlebars file which should be unedited.  :)

**Comment by monikamaheshwari** [comment]
why not to remove `<div class="panel-section-tabs-separator"></div>`?

**Comment by april** [comment]
I'm not sure what you're talking about? If you look at the changes, you can see that the only thing that has changed is a carriage return at the end of the file. This shouldn't be in the commits.  :)

**Comment by april** [comment]
(updated my comment above)

**Comment by monikamaheshwari** [comment]
Oh yes I don't know why atom is adding that.

**Comment by april** [comment]
Now you have a different newline at the end of index.css.  :)

File: src/viewer/css/index.css [hunk 1]
```diff
@@ -143,6 +143,8 @@ h2 {
   height: auto;
   justify-content: center;
   margin: 0;  /* default 0 -1 causes x overflow */
+  border: 1px solid rgba(0, 0, 0, 0.1);
```
Comment (april) [inline src/viewer/css/index.css]: In my testing, this doubles the thickness of the border on the bottom.  :)

**Comment by april** [comment]
Merged in 67c60309f9b7152ba8d77e66eb4b46572bce38af.

**Comment by monikamaheshwari** [comment]
Thanks :)

**Pull request closed without merging**